### PR TITLE
ENG-11432 check various stats for shutdown and pause

### PR DESCRIPTION
**Pull request by zeemanhe**
graceful shutdown: 
wait for in-flight txns, importer, dr consumer, async cmdlog txns fsynced.

shutdown --save and voltadmin pause --wait: 
wait for in-flight txns, importer, dr consumer, async cmdlog txns fsynced, exporter, and dr producer.

### CODE REVIEW

File: lib/python/voltcli/voltadmin.d/shutdown.py [hunk 1]
```diff
@@ -44,18 +44,19 @@ def shutdown(runner):
...
             if runner.opts.save:
                columns = [VOLT.FastSerializer.VOLTTYPE_BIGINT]
                shutdown_params =  [zk_pause_txnid]
+               #save option, check more stats
+               checkstats.check_exporter(runner)
+               checkstats.check_dr_producer(runner)
         except (KeyboardInterrupt, SystemExit):
             runner.info('The cluster shutdown process has stopped. The cluster is still in a paused state.')
             runner.abort('You may shutdown the cluster with the "voltadmin shutdown --force" command, or continue to wait with "voltadmin shutdown".')
+    runner.info('Cluster is ready for shutdown')
```
Comment (xinjiacs) [inline lib/python/voltcli/voltadmin.d/shutdown.py]: This message is only printed when doing non-force shutdown before. So you should move it inside the IF case.

Also the message should be something: "Cluster is safe to shutdown, everything has been drained."

File: lib/python/voltcli/voltadmin.d/pause.py [hunk 2]
```diff
@@ -27,7 +27,7 @@
     )
 )
 def pause(runner):
-    # Check the STATUS column. runner.call_proc() detects and aborts on errors.
+   # Check the STATUS column. runner.call_proc() detects and aborts on errors.
```
Comment (xinjiacs) [inline lib/python/voltcli/voltadmin.d/pause.py]: on purpose?

File: lib/python/voltcli/voltadmin.d/pause.py [hunk 3]
```diff
@@ -39,5 +39,9 @@ def pause(runner):
             runner.error('The cluster has failed to quiesce with status: %d' % status)
             return
         runner.info('The cluster is quiesced.')
-        #check the dr stats
-        checkstats.check_export_dr(runner)
+        checkstats.check_clients(runner)
```
Comment (xinjiacs) [inline lib/python/voltcli/voltadmin.d/pause.py]: Can you do similar catch for CTRL-C like shutdown?

Comment (zeemanhe) [inline lib/python/voltcli/voltadmin.d/pause.py]: ctrl-c handling was added

File: lib/python/voltcli/voltadmin.d/shutdown.py (hunk 1 above)
Comment (zeemanhe) [inline lib/python/voltcli/voltadmin.d/shutdown.py]: updated.

**Pull request merged**